agent/util-scripts/gold/pbench-clear-tools/test-68.txt

```diff
@@ -1,5 +1,6 @@
 +++ Running test-68 pbench-clear-tools --name=vmstat --remotes=fubar5.example.com,fubar6.example.com
 Removed "vmstat" from host, "fubar5.example.com", in tools group, "default"
+Tool vmstat not registered for remote fubar6.example.com
```

Comment:
Can we drop this message on the console, and just leave it as a "debug" message only?

Reply:
fixed